### DISCUSSION TITLE
Install packages behind a company proxy

### DISCUSSION THREAD

**Discussion by incoggnito** [python-poetry/poetry#3700] [Q&A]
I thought poetry would just wrap pip...
why don't the proxy settings in `pip.ini` work directly for poetry?

**Comment by incoggnito** [comment]
Ah ok, that's because of the requests package. The environment variables within the .bashrc config in windows does not work for me.
I use two different win10 environments. One behind a company proxy and with a private package server, the other without this restrictions.
This config works great at the restricted environment.
```
[[tool.poetry.source]]
name = "myserver"
url = "https://nexus.myserver.net/repository/pypi/simple/"
default = true
```

Can this configuration be turned off with git hooks, or in some other way?
Maybe a seperat config file that i could just add to the gitignore file.

**Reply by andreas-vester** [reply]
I am stuck with the same problem. I have to use a private package server and I am using the same ``[[tool.poetry.source]]`` configuration in my ``pyproject.toml``.
However, I can't commit it to GitHub as I am rightly not allowed to reveal any internal server names. Besides, this configuration isn't of any use to other users who might be willing to contribute to the project.

**A seperate config file would be probably the best solution. I already opened an [issue](https://github.com/python-poetry/poetry/issues/5958).**

In the meantime, I need to figure out a workaround. Automatically removing this particular section before commiting and re-inserting afterwards would be a potential way to go.

Any ideas?
